workflow/workflow_commands.go

```diff
@@ -54,6 +54,9 @@ var (
 		"Signal": enumspb.RESET_REAPPLY_TYPE_SIGNAL,
 		"None":   enumspb.RESET_REAPPLY_TYPE_NONE,
 	}
+	updateWaitPolicyMap = map[string]interface{}{
+		"Completed": enumspb.UPDATE_WORKFLOW_EXECUTION_LIFECYCLE_STAGE_COMPLETED,
```

Comment:
Is it standard practice for us to capitalize these tokens? I would have expected this to be all lowercase but if this is the common practice then it's fine. Alternatively we could put both spellings here.

Reply:
That's a very good question, I followed the implementation right above this that uses camel case and feeds it directly to a switch block where it is case sensitive. I am not sure if we have standard practice though. Let me follow up on this and if there is one, leave a comment here to make it clear, and if we don't have one have a discussion around it to come up with one.